examples/nextjs/src/app/[[...slug]]/page.js

```diff
@@ -4,45 +4,66 @@ import { MyPage } from "@/components/my-page";
 const client = dotcmsClient.init({
     dotcmsUrl: process.env.NEXT_PUBLIC_DOTCMS_HOST,
     authToken: process.env.DOTCMS_AUTH_TOKEN,
-    siteId: '59bb8831-6706-4589-9ca0-ff74016e02b2',
+    siteId: "59bb8831-6706-4589-9ca0-ff74016e02b2",
     requestOptions: {
         // In production you might want to deal with this differently
-        cache: 'no-cache'
-    }
+        cache: "no-cache",
+    },
 });
 
 export async function generateMetadata({ params, searchParams }) {
-
     const requestData = {
-        path: params?.slug ? params.slug.join('/') : 'index',
+        path: params?.slug ? params.slug.join("/") : "index",
         language_id: searchParams.language_id,
-        'com.dotmarketing.persona.id': searchParams['com.dotmarketing.persona.id'] || '',
+        "com.dotmarketing.persona.id":
+            searchParams["com.dotmarketing.persona.id"] || "",
         mode: searchParams.mode,
-        variantName: searchParams['variantName']
+        variantName: searchParams["variantName"],
     };
 
     const data = await client.page.get(requestData);
 
     return {
-        title: data.entity.page.friendlyName || data.entity.page.title
+        title: data.entity.page.friendlyName || data.entity.page.title,
     };
 }
 
 export default async function Home({ searchParams, params }) {
     const requestData = {
-        path: params?.slug ? params.slug.join('/') : 'index',
+        path: params?.slug ? params.slug.join("/") : "index",
         language_id: searchParams.language_id,
-        'com.dotmarketing.persona.id': searchParams['com.dotmarketing.persona.id'] || '',
+        "com.dotmarketing.persona.id":
+            searchParams["com.dotmarketing.persona.id"] || "",
         mode: searchParams.mode,
-        variantName: searchParams['variantName']
+        variantName: searchParams["variantName"],
     };
 
     const data = await client.page.get(requestData);
 
+    const data2 = await client.content
```

Comment:
A good example for this is to have this in client side which brings new issues:

1. A new client instance, which is fine because we don't have state in the client instance
2. Do we need the auth token in the client side? I would say our example can be with content that can be accesible without auth
3. If 2 is true, does that mean that our client can be init without auth token?
4. Is 3 is true... how do we habdle errors when usera try to access an API that actually needs auth token?

Questions, questions questions.

Reply:
We can prevent the access to methods or parts of the API that requires an Auth Token, that should not be a problem